bungiesearch/management/commands/search_index.py

```diff
@@ -136,7 +136,8 @@ def handle(self, *args, **options):
 
                 logging.info('Creating index {} with {} doctypes.'.format(index, len(mapping)))
                 es.indices.create(index=index, body={'mappings': mapping, 'settings': {'analysis': analysis}})
-                es.cluster.health(index=index, wait_for_status='yellow')
+
+            es.cluster.health(index=','.join(indices), wait_for_status='green', timeout=30)
```

Comment:
This is much more efficient waiting, we can wait on all of the indices in parallel now.